lib/network/kmip/index.ts

```diff
@@ -28,6 +28,23 @@ function _PrimitiveType(tagName: string, type: string, value: any) {
     return { [tagName]: { type, value } };
 }
 
+/** Thales has a 24h idle tls session error. We can close and reopen new connection */
```

Comment:
worth commenting the Thales version so future devs know if something has changed and documentation link